### DISCUSSION TITLE
After around 600 faces no more face recognition takes place

### DISCUSSION THREAD

**Discussion by Kruegerbei** [nextcloud/recognize#774] [Q&A] [answered]
Hello there, got a problem where recognize stops working: 
So I recently uploaded around 10000 Photos to my nextcloud and started  with face and object recognition. The first few hundred worked quite flawlessly apart of some clusters which were to big and thus quite inaccurate. After a while I started noticing that no new faces and photos were added to the people tab, while the objekt tab grew. Also the face queue was way to short. After several resets of faces and clusters I ran today occ recognize:classify and got following output on every single classification run:
Face score too low. continuing with next face. or
Face is not straight. continuing with next face.
No more information in the logs sadly.
Anyone experiencing the same issues?

Which version of recognize are you using?
3.7.0

Enabled Modes
Object recognition, Face recognition,

TensorFlow mode
Normal mode

Which Nextcloud version do you have installed?
25.0.0

Which Operating system do you have installed?
Ubuntu 22.04 Server

Which Docker container are you using to run Nextcloud? (if applicable)
N/A

How much RAM does your server have?
8 GB

What processor Architecture does your CPU have?
X86_64

**Comment by marcelklehr** [comment, marked as answer]
A lot of issues have been resolved in the meantime with the scheduling of classifier processes. I suggest you have at it again and file a bug report if it still doesn't work :)
